platforms/osx/src/main.mm

```diff
@@ -13,11 +13,9 @@
 
 std::string sceneFile = "scene.yaml";
 
-std::string markerStyling = "{ style: 'points', interactive: true, color: 'white', size: [25px, 25px], order: 100, collide: false }";
+std::string markerStyling = "touch.point.icons";
```

Comment:
Hmm will update. I would like to get some opinion on the user of our API to make sure I am not the only one who feel this is redundant information being asked from the user.

cc. @ecgreb 

But update coming.